IBM/openapi-validator
# feat: error for parameter without schema or content

Purpose:
- OAS3 schema requires that either a schema or a content object must be defined for any parameter. Tooling based on OAS3 will expect parameters to have a schema or content.

Changes:
- Add check that a schema is provided in content objects for parameters.
- Check that schema or content object provided for all parameters.

Test:
- Add test to ensure that error returned for parameter with no schema and no content objects.
- Add test to ensure a warning given for a content object in a parameter with no schema object.

Docs:
- Document the parameter-contains-schema rule.
